barserver/src/index.ts

```diff
@@ -10,6 +10,11 @@ app.get( "/", ( req, res ) => {
     res.send( "Hello world!" );
 } );
 
+app.post("/api/postsecret",(req,res)=>{
+    const secret=req.body.data;
+    res.send("secret");
```

Comment:
这里就只是send了一个字符串？是不是应该
```suggestion
    res.send(secret);
```